src/tabletraits.jl

```diff
@@ -1,66 +1,48 @@
-struct DESolutionIterator{T,S}
-    sol::S
+Tables.isrowtable(::Type{T}) where {T} = true
```

Comment:
Is this piracy?

Reply:
Oops! Yeah, this is supposed to be
```suggestion
Tables.isrowtable(::Type{<:DESolution}) = true
```